### ISSUE TITLE
Fields we can probably remove

### ISSUE THREAD

**Issue by wmandrews** [newsdev/elex-loader#40]
Here are the fields we never used or no longer have use for.

`nyt_race_preview`
`nyt_delegate_allocation`
`nyt_race_name`
`nyt_race_result_description`
`nyt_called`
`nyt_race_important`
`nyt_candidate_description`
`nyt_delegates`
`nyt_races`*
`race_raceid`*
`race_postal`*

*Was never used by the election-2016 app, but there could be other uses.

**Comment by jeremyjbowers** [comment]
I use `nyt_races` and we need `nyt_delegate_allocation` for primaries.

**Comment by wmandrews** [comment]
You're right about `nyt_delegate_allocation`, though none of our currently live pages are using it (maybe it was dropped from an overrides file at some point and never made it back in).
